packages/generator-single-spa/src/react/generator-single-spa-react.js

```diff
@@ -202,12 +202,12 @@ module.exports = class SingleSpaReactGenerator extends Generator {
         coloredFinalInstructions(
           `1. Run '${this.options.packageManager} start${
             this.options.packageManager === "npm" ? " --" : ""
-          } --https --port 8500'`
```

Comment:
SSL certs pwn our users pretty frequently. The single-spa-playground intentionally works on http so that users don't have to worry about it. @Sauloxd and i discussed this change and i think it would be a good one to lower friction.